std/assembly/typedarray.ts

```diff
@@ -95,6 +95,10 @@ export class Int8Array extends ArrayBufferView {
   reverse(): this {
     return REVERSE<this, i8>(this);
   }
+
+  static wrap(buffer: ArrayBuffer): Int8Array {
```

Comment:
I wonder why `WRAP` marco exposed `byteOffset` and `length`, but actual `wrap` is not?
Probably:
```ts
static wrap(buffer: ArrayBuffer, byteOffset: i32 = 0, length: i32 = -1): Int8Array {
```
will be more convenient

Reply:
Why `length: i32 = -1` instead of `length: i32 = buffer.byteLength`?

Reply:
`length = buffer.byteLength` would only be valid in the Uint8Array/Int8Array case. In Uint16Array for example it would have to be `buffer.byteLength / 2`. Maybe `length: i32 = buffer.byteLength >>> alignof<T>()` (if that works)?

Reply:
I see. Fixed